### PR TITLE
add product card

### PR DESCRIPTION
**Pull request by MartaChobaniuk**
- [DEMO LINK](https://martachobaniuk.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://martachobaniuk.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  line-height: 18px;
+}
+
+.product-card__price-name {
+  font-size: 12px;
+  color: #616070;
+}
+
+.product-card__price-value {
+  font-size: 16px;
```
Comment (YegorKorenkov) [inline src/style.css:87]: font-size 16px by default, please remove

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+  line-height: 18px;
+}
+
+.product-card__price-name {
```
Comment (YegorKorenkov) [inline src/style.css:81]: Als otry to reset default margin 
image

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  font-size: 12px;
+  color: #616070;
+}
+
+.product-card__price-value {
```
Comment (YegorKorenkov) [inline src/style.css:86]: The same

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #fff;
+  background-color: #00acdc;
+  border-radius: 5px;
+}
+
+.product-card__button:hover {
+  background: #fff;
+  border: 1px solid #00acdc;
```
Comment (YegorKorenkov) [inline src/style.css:113]: You alredy add rule border for .product-card__button, change only color

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+  border-radius: 5px;
```
Comment (YegorKorenkov) [inline src/style.css:114]: the same

**Review by YegorKorenkov — CHANGES_REQUESTED** [review]
Please fix and re-run tests

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,69 @@
...
+    >
+      <div class="product-card__image-container">
+        <img
+        class="product-card__image"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+      </div>
+      <div class="product-card__name">
+        APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
```
Comment (vlad-demchuk) [inline src/index.html:37]: you forgot  `Monoblock`
image

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.product-card {
```
Comment (vlad-demchuk) [inline src/style.css:11]: image
create files for bem blocks

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 24px;
+}
+
+.product-card__price-name {
+  margin: 0;
```
Comment (vlad-demchuk) [inline src/style.css:83]: you could create a separate file for reset as well and reset styles for some special tags you need

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**